src/lua/msgpack.c

```diff
@@ -288,8 +315,17 @@ luamp_encode_r(struct lua_State *L, struct luaL_serializer *cfg,
 }
 
 enum mp_type
-luamp_encode(struct lua_State *L, struct luaL_serializer *cfg,
-	     struct mpstream *stream, int index)
+luamp_encode_r(struct lua_State *L, struct luaL_serializer *cfg,
+	       struct mpstream *stream, struct luaL_field *field, int level)
+{
+	return luamp_encode_with_translation_r(L, cfg, stream, field,
```

Comment:
Nit: Could be static inline and reside it in the header IMO. The comment wouldn't be necessary then. Same comment for `luamp_encode`.